app/retail/views.py

```diff
@@ -1574,3 +1575,39 @@ def clean_url(url):
     }
 
     return TemplateResponse(request, 'admin_index.html', context)
+
+
+@cached_as(JSONStore.objects.filter(view='posts', key='posts'), timeout=1200)
+@cached_view
+def fetchPost(qt='2'):
+    jsonstores = JSONStore.objects.filter(view='posts', key='posts')
+    if jsonstores.exists():
+        return jsonstores.first().data
+
+
+@cached_view(timeout=86400) # cached for 24 hours
```

Comment:
maybe is better to name the cached object so if we need to invalidate it later will be easier